Aptoide/aptoide-client-v8
# [ASV-1400] Initialize flurry

**What does this PR do?**

   This PR aims at increasing flurry sdk versions to attempt at fixing problems regarding the ads. This was recommended by Flurry itself.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] aptoide-client-v8

**How should this be manually tested?**

  You should check if you have ads from flurry, but currently this is not possible, since there are changes being performed on the mopub dashboard. We should check this part on the nightly.
Also check if the events associated with Flurry, for example Download Complete.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1400](https://aptoide.atlassian.net/browse/ASV-1400)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 


**Code Review Checklist**
